### PR TITLE
Updated styling for ArgDescriptor component in workspaces/tests #146

### PR DESCRIPTION
**Pull request by Subham5401**
Title: Modified "workspaces/docs/src/components/ArgDescriptor/styles.module.css"

Description:
I've made some changes to the mentioned file to fit the ask. 

Changes Made:
- Added bottom margin of 10px to .arg-description--name and .arg-description--type
- Added bottom margin, border bottom and padding bottom as mentioned for .description

Please review and let me know if there are any further changes or suggestions. I'm open to feedback and willing to make additional updates as needed. Thanks for considering this pull request!

### CODE REVIEW

**Comment by Subham5401** [comment]
Committed a few more changes:
- Ignore Storybook in workspaces/calendar-widgets/.npmignore
- Remove Default FavIcon from Docs

File: workspaces/docs/docusaurus.config.js [hunk 1]
```diff
@@ -1,57 +1,57 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
```
Comment (9mbs) [inline workspaces/docs/docusaurus.config.js:4]: We're introducing a-lot of unnecessary changes in this file, I believe changes in this file should only be scoped to #149

File: workspaces/docs/src/components/ArgDescriptor/styles.module.css [hunk 2]
```diff
@@ -18,4 +19,14 @@
...
   color: var(--secondary-text-color);
-}
\ No newline at end of file
+}
+
+.arg-description--type {
+  margin-bottom: 10px;
+}
+
+.description {
```
Comment (9mbs) [inline workspaces/docs/src/components/ArgDescriptor/styles.module.css:28]: Can we update the class name on this component to `.arg-description--description` 

Its verbose I know, but it helps things remain consistent 🙂

**Comment by 9mbs** [comment]
Hi @Subham5401 how awesome, thank you for taking on so many issues! 💯 Moving forward, i'd kindly request we keep each PR consolidated to 1 issue so we can avoid scope-creeping in merges. In this case, these are all relatively small changes with little chance of regressions so there are no worries. 🙂 

I left a few comments, please let me know if you have any questions! Awesome work btw!

File: workspaces/docs/docusaurus.config.js [hunk 3]
```diff
@@ -1,57 +1,57 @@
...
 const config = {
-  title: 'calendar-widgets',
+  title: "calendar-widgets",
   tagline:
-    'A collection of calendar widgets and utilities to make working with calendars less shitty',
-  favicon: 'img/favicon.ico',
-  url: 'https://calendar-widgets.com',
-  baseUrl: '/',
+    "A collection of calendar widgets and utilities to make working with calendars less shitty",
+  favicon: "",
```
Comment (9mbs) [inline workspaces/docs/docusaurus.config.js:12]: it looks like this being an empty string is causing the Docusaurus build to fail. Can we remove this field entirely? https://docusaurus.io/docs/api/docusaurus-config#favicon

File: workspaces/docs/src/components/ArgDescriptor/styles.module.css (hunk 2 above)
Comment (Subham5401) [reply workspaces/docs/src/components/ArgDescriptor/styles.module.css:28]: Yeah sure.

File: workspaces/docs/docusaurus.config.js (hunk 3 above)
Comment (Subham5401) [reply workspaces/docs/docusaurus.config.js:12]: Hmm, I see.
Will check that.

File: workspaces/docs/docusaurus.config.js (hunk 1 above)
Comment (Subham5401) [reply workspaces/docs/docusaurus.config.js:4]: I think the code formatting tool I use is reducing indentation spaces, which is watched by git. I'll take care of this.

Comment (Subham5401) [reply workspaces/docs/docusaurus.config.js:4]: Shall I undo it and push a commit?

Comment (9mbs) [reply workspaces/docs/docusaurus.config.js:4]: That's okay, we can leave it as it. I'll need to get some ESLint rules in this directory to help with code styles but that will come at another time. 🙂

**Comment by 9mbs** [comment]
These changes are looking great @Subham5401 ! I really appreciate you taking the time contribute to the project. You've covered a-lot of ground with this one. 💯 I'm going to merge these changes into the `main` branch now.

**Comment by Subham5401** [comment]
I'm grateful for the opportunity. Thank you @9mbs !
